packages/dapi-grpc/Cargo.toml

```diff
@@ -10,7 +10,8 @@ authors = [
     "Ivan Shumkov <shumkov@dash.org>",
 ]
 edition = "2021"
-rust-version = "1.73"
+# Update using scripts/update-rust-toolchain.sh
+rust-version = "1.76"
```

Comment:
Why we set it here?

Reply:
I don't understand the question.

The rust-version field is an optional key that tells cargo what version of the Rust language and compiler your package can be compiled with. 

As we only test on 1.76, we only support this version onwards.

Reply:
Yeah, I've never seen such an option, and since it's a minimal value then it's all good.